examples/Atlas/sdf/cinder_block_wide/model.sdf

```diff
@@ -17,104 +17,23 @@
       </inertial>
```

Comment:
are these all going to collide with @scottviteri 's pull request that just got merged?